### ISSUE TITLE
Cannot connect to already running jackd

### ISSUE THREAD

**Issue by ana0** [supercollider/supercollider#4551]
Hey, new to supercollider!

I'm on xubuntu 16.04, finally got jackd working, I can start it like this `sudo jackd -d alsa -d hw:1` - but I can't get the supercollider server to connect to it. Have spent some time trying to get jackd to start up with `hw:0` but that path has proven fruitless so far

`supernova -H hw:1` gives 

```
Supernova booting
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for 4294967295, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for 4294967295, skipping unlock
Error: Unable to connect to JACK server
```

And trying to boot the server in the ide, where you can see it's still using `hw:0`:

```
booting 57110
jackdmp 1.9.11
Copyright 2001-2005 Paul Davis and others.
Copyright 2004-2014 Grame.
jackdmp comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome to redistribute it
under certain conditions; see the file COPYING for details
JACK server starting in realtime mode with priority 10
self-connect-mode is "Don't restrict self connect requests"
audio_reservation_init
Acquire audio card Audio0
creating alsa driver ... hw:0|hw:0|1024|2|48000|0|0|nomon|swmeter|-|32bit
could not initialize audio.
RESULT = 1
ERROR: server failed to start
```
Info about `hw:0` and `hw:1`:

```
cat /proc/asound/cards
 0 [HDMI           ]: HDA-Intel - HDA Intel HDMI
                      HDA Intel HDMI at 0xf7e34000 irq 48
 1 [PCH            ]: HDA-Intel - HDA Intel PCH
                      HDA Intel PCH at 0xf7e30000 irq 47
```

Any help appreciated!

**Comment by jamshark70** [comment]
I can reproduce this problem if I run JACK as superuser and SC as a normal user.

I have no idea how JACK handles multi-user scenarios, but I've never heard of anyone running JACK as superuser and it wouldn't surprise me at all if JACK client apps could connect only to a JACK server running under the same user.

TL;DR don't use `sudo`.

I'm going to take a wild guess that you ran into some user privilege issue, probably "cannot use real-time scheduling (FIFO at priority 10)." Standard Linux audio advice is to add your (normal) user to the audio group.

http://www.jackaudio.org/faq/linux_rt_config.html

In any case, this is definitely a JACK configuration issue and not a SuperCollider bug.

**Comment by ana0** [comment]
Yes, running without sudo gave so many `Operation not permitted (1)`s that I didn't think it was working.  Seems to be doing ok now - closing and thanks!
